src/javascripts/crosswords/crossword.js

```diff
@@ -698,7 +698,12 @@ class Crossword extends Component {
   }
 
   hasSolutions() {
-    return 'solution' in this.props.data.entries[0];
+    const dateSolutionAvailable = this.props.data.dateSolutionAvailable;
```

Comment:
I would use destructuring assignment on those deep objects just to make things a little more readable